ODIN_II/regression_test/benchmark/verilog/keywords/inout/double_assign_inout_input

```diff
@@ -0,0 +1,7 @@
+GLOBAL_SIM_BASE_CLK en in_out1 in_out2
+0 1 z z
```

Comment:
We cannot do 'z' on primary inputs (PI) and PO, I'm working on it right now for internal nodes, but IDK that on the top ports it would be a thing, but you make a good point here.
Adding z to PI and PO is not an invalid idea and we stay backward compatible, I'm just not sure If I wanted to expose it, I have no idea who came up with the _input and _output vector standard and if its something already documented, we should stick to that. But so far I have no clue if there is any standards for it.
 @djns99

Reply:
alright! figured out a solution I think...

http://www.pldworld.com/_hdl/2/_ref/se_html/manual_html/c_vcd10.html

http://www.pldworld.com/_hdl/2/_ref/se_html/manual_html/c_vcd11.html

http://www.pldworld.com/_hdl/2/_ref/se_html/manual_html/c_vcd12.html